### PR TITLE
Fixes for comments and playlist articles

### PR DESCRIPTION
**Pull request by ahilles107**

### CODE REVIEW

File: newscoop/src/Newscoop/NewscoopBundle/Controller/CommentsController.php [hunk 1]
```diff
@@ -298,11 +299,11 @@ public function setRecommendedAction(Request $request, $comments, $recommended)
                 continue;
             }
             $comment = $em->getRepository('Newscoop\Entity\Comment')->find($commentId);
-
+            $thread = $em->getRepository('Newscoop\Entity\Article')->findOneBy(array('number' => $comment->getThread()));
```
Comment (m038) [inline newscoop/src/Newscoop/NewscoopBundle/Controller/CommentsController.php]: Shouldn't we add a checl if $thread == null?

**Comment by m038** [comment]
:+1:

File: newscoop/src/Newscoop/NewscoopBundle/Controller/CommentsController.php (hunk 1 above)
Comment (ahilles107) [inline newscoop/src/Newscoop/NewscoopBundle/Controller/CommentsController.php]: hmm... thread is required - you can't create comment without article attached, and on removing articles we remove also comments - i tested it so in my opinion - check is not needed - it works like it worked before (before it was lazy loaded based on relation).

Comment (m038) [inline newscoop/src/Newscoop/NewscoopBundle/Controller/CommentsController.php]: ok

**Comment by takeit** [comment]
:+1:

**Pull request merged**